### PR TITLE
bot: refresh admins list

### PR DESCRIPTION
**Pull request by refack**
Removed inactive contributors, added myself.
Needs follow up:
* GitHub Team page https://github.com/orgs/nodejs/teams/github-bot/members
* GPG Access in secrets repo (maybe cycle the keys)

/CC @joaocgreis @mhdawson @rvagg @joyeecheung @nodejs/github-bot @nodejs/build-infra

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,14 +63,9 @@ missing please open an issue. If you are interested in joining check out
 
 <!-- ncu-team-sync.team(nodejs/github-bot) -->
 
-- [@Fishrock123](https://github.com/Fishrock123) - Jeremiah Senkpiel
-- [@jbergstroem](https://github.com/jbergstroem) - Johan Bergström
 - [@joyeecheung](https://github.com/joyeecheung) - Joyee Cheung
-- [@maclover7](https://github.com/maclover7) - Jon Moss
```
Comment (phillipj) [inline README.md]: @maclover7 has been responding to several bot PRs quicker than me this summer/fall. Have you double checked he's not interested in contributing anymore?

Comment (refack) [reply README.md]: IIUC he started college, and has not been active since september. When he will become active again we should restore his access.

Comment (phillipj) [reply README.md]: Alrighty, thanks for elaborating 👍

**Comment by richardlau** [comment]
> Removed inactive contributors, added myself.
> Needs follow up:
> 
>     * GitHub Team page https://github.com/orgs/nodejs/teams/github-bot/members
> 

Maybe it would be better to update the GitHub team first and then resync the list using [`ncu-team sync`](https://github.com/nodejs/node-core-utils/blob/master/docs/ncu-team.md) (the markers were added in https://github.com/nodejs/build/pull/1108)?

**Comment by refack** [comment]
> Maybe it would be better to update the GitHub team first and then resync the list using [`ncu-team sync`](https://github.com/nodejs/node-core-utils/blob/master/docs/ncu-team.md) (the markers were added in #1108)?

Sure, this PR is mostly symbolic, since GPG signatures are needed to be added to the secrets, and as you mentioned need to update the team.

BTW: @richardlau I see your active in this domain, would you like to be added?

**Comment by maclover7** [comment]
> @maclover7 has been responding to several bot PRs quicker than me this summer/fall. Have you double checked he's not interested in contributing anymore?

Just want to note that when I had looked at moving some Build WG folks to "emeritus" status, I would first email them to see if they were interesting in contributing, and, if not, _then_ removing them. I didn't get any such notification related to this PR...

**Comment by refack** [comment]
> I didn't get any such notification related to this PR...

I'm sorry for the omission @maclover7. If you want to stay active with bot maintenance, I'd be happy to amend my suggestion.

FTR: This does not relate to WG emeritus status, just a refresh of a specific resource access.

**Comment by richardlau** [comment]
> 
> 
> > Maybe it would be better to update the GitHub team first and then resync the list using [`ncu-team sync`](https://github.com/nodejs/node-core-utils/blob/master/docs/ncu-team.md) (the markers were added in #1108)?
> 
> Sure, this PR is mostly symbolic, since GPG signatures are needed to be added to the secrets, and as you mentioned need to update the team.
> 
> BTW: @richardlau I see you're active in this domain, would you like to be added?

Which domain are you referring to?

**Comment by refack** [comment]
> Which domain are you referring to?

The GitHub bot? (although I assume the whole BuildWG would be happy for you assistance)

**Comment by richardlau** [comment]
> 
> 
> > Which domain are you referring to?
> 
> The GitHub bot? (although I assume the whole BuildWG would be happy for you assistance)

I'm certainly interested in the bot and it's on my list of things to look into. It's probably a bit too early for me to be administering it. I'm not sure I'd currently describe myself as active -- I think I've only ever commented in a single issue in https://github.com/nodejs/github-bot, for example.

**Comment by sam-github** [comment]
Stale PR, README was updated elsewhere.

**Pull request closed without merging**